SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Storage/Storage.cs

```diff
@@ -89,8 +82,11 @@ public Task PutLog(string channelName, Log log)
             return AddTaskToQueue(() =>
             {
                 var logJsonString = LogSerializer.Serialize(log);
-                var logEntry = new LogEntry { Channel = channelName, Log = logJsonString };
-                _storageAdapter.InsertAsync(logEntry).GetAwaiter().GetResult();
+                _storageAdapter.Insert(TableName,
+                    new[] { ColumnChannelName, ColumnLogName },
+                    new List<object[]> {
```

Comment:
do we actually insert non string values? we should change type in interface.

Reply:
Yes - integers.

> we should change type in interface.

??

Reply:
To array of strings? No.

Reply:
https://github.com/microsoft/appcenter-sdk-dotnet/pull/1246#discussion_r355109886